src/app/containers/Headings/index.jsx

```diff
@@ -17,6 +18,9 @@ const StyledHeadline = styled(Headline)`
   :focus {
     outline: none;
   }
+  ${MEDIA_QUERY_TYPOGRAPHY.LAPTOP_AND_LARGER} {
+    word-break: break-word;
+  }
```

Comment:
- Although the layout isn't _as_ visibly broken below `LAPTOP_AND_LARGE`, it is still broken (see the horizontal scrollbar) and therefore this fix probably shouldn't be scoped to a specific screen size.
- `word-break: break-word` has been deprecated, and also means we would break words even if allowing the browser to put them on a new line would prevent overflow. `overflow-wrap` might be helpful here instead.

Can we remove the media query and try `overflow-wrap: anywhere;`?

Any change to this component will be far reaching, so we should triple check for regressions - across multiple services, screen sizes and browsers.

Reply:
I have manually checked through the STYs on local from https://github.com/bbc/simorgh/blob/latest/docs/Simorgh-Release-Info.md and some of the PGLs where there is no STY on local. 

`overflow-wrap: anywhere` does not seem to cause any visual regressions when the viewport width is above 300px (a narrow phone has a width of 320px) so it seems that it will be very unlikely for a word to be so long that it does not fit on a single line in a headline. 

Some languages eg. Zhongwen have different rules for starting a new line https://devblogs.microsoft.com/oldnewthing/20160307-00/?p=93122 and with `overflow-wrap: anywhere` the behaviour of the word wrapping stays as it is currently.

For very small viewport widths, around the 260px mark, the behaviour changes and a word will be split over two lines, eg. http://localhost:7080/tamil/global-47758688, though the current behaviour at this viewport width is for the user to have to scroll horizontally, so I think the regression is acceptable and furthermore it is rare to look at a webpage at this very narrow width. 

So I don't feel we need to scope the change to certain problematic services, though realise that not knowing the expected behaviour of all the services regarding where to start a new line, it is hard to be completely sure that problems are not being caused.

This [article](https://www.w3.org/International/articles/article-text-size.en) refers to translation but has some interesting points about the varying width of different languages.

Reply:
Thanks, Simon. The links were really handy.
It sounds like proceeding with `overflow-wrap: anywhere;` is the way to go, and then to keep an eye on things.